modules/data-loading/pages/data-streaming-connector.adoc

```diff
@@ -400,12 +332,293 @@ GSQL > RUN LOADING JOB load_person
 By default, loading jobs that use Kafka data sources run in streaming mode and do not stop until manually aborted.
 As data is streamed from the data source, the running loading job will continuously ingest the streamed data into the graph store.
 
-==== Stream data source updates
+=== Stream data source updates
 
 To load updates to the data source after you've created the connectors, you need to delete the connector and recreate another connector.
 
 In the future, the streaming data connector will support automatically scanning for updates and stream data to TigerGraph.
 
+== Stream data from an external Kafka cluster
+You can create a data connector between an external Kafka cluster (source cluster) and TigerGraph’s internal Kafka server (destination cluster) to stream data from the source cluster.
+The connector streams the Avro data to TigerGraph's internal Kafka server, and transforms the data into JSON format.
```

Comment:
The avro data transformation is done with our Avro converter which is a different component from our connectors. If users have csv in their external kafka, they can change the avro converter to bytearray converter and the csv format will remain and data will be streamed to the target kafka in csv.